doc/regression-tests.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-Unusually this project includes a set of regression tests. These tests are not *normally* run as part of the `default` Rake task, because they take a long time to run. Thus they do not normally run as part of the [main CI build](continuous-integration#main). However, they *always* run as part of the [regression CI build](continuous-integration#regression) on the master branch only. Go to this [Jenkins Job](https://ci.integration.publishing.service.gov.uk/job/smartanswers/job/master/), click on the lastest build, click replay and edit the script in the first textarea to run the regressions tests. 
+Unusually this project includes a set of regression tests. These tests are not *normally* run as part of the `default` Rake task, because they take a long time to run. Thus they do not normally run as part of the [main CI build](continuous-integration#main). However, they *always* run as part of the [regression CI build](continuous-integration#regression) on the master branch only. Go to this [Jenkins Job](https://ci.integration.publishing.service.gov.uk/job/smartanswers/job/master/), click on the lastest build, click replay and edit the script in the first textarea to run the regressions tests.
 
```

Comment:
The replay part has since change.. all that is need is selecting the regression test button to enable and run regression tests as part of a ci build.